hal/src/main/native/sim/mockdata/PCMData.cpp

```diff
@@ -49,6 +49,23 @@ DEFINE_CAPI(HAL_Bool, ClosedLoopEnabled, closedLoopEnabled)
 DEFINE_CAPI(HAL_Bool, PressureSwitch, pressureSwitch)
 DEFINE_CAPI(double, CompressorCurrent, compressorCurrent)
 
+void HALSIM_GetPCMAllSolenoids(int32_t index, uint8_t* values) {
+  auto& data = SimPCMData[index].solenoidOutput;
+  uint8_t ret = 0;
+  for (int i = kNumSolenoidChannels - 1; i >= 0; i--) {
```

Comment:
Why is this a reverse loop?

Reply:
Because I'm shifting it into another, although I could go forward and shift the value to insert rather then the output value.